easier_net/easier_net.py

```diff
@@ -45,94 +40,68 @@ def __init__(
         self.full_tree_pen=full_tree_pen
         self.input_pen=input_pen
         self.num_batches=num_batches
-        self.n_jobs=n_jobs
```

Comment:
i'd put a note somewhere that we want to put back `n_jobs` as an option eventually. this argument would let us fit multiple SierNets in parallel. we can implement this part later. ;)